pkg/descheduler/strategies/node_affinity_test.go

```diff
@@ -215,6 +216,11 @@ func TestRemovePodsViolatingNodeAffinity(t *testing.T) {
 			sharedInformerFactory.Start(ctx.Done())
 			sharedInformerFactory.WaitForCacheSync(ctx.Done())
 
+			eventBroadcaster := events.NewEventBroadcasterAdapter(fakeClient)
+			eventBroadcaster.StartRecordingToSink(ctx.Done())
+			eventRecorder := eventBroadcaster.NewRecorder("sigs.k8s.io.descheduler")
+			defer eventBroadcaster.Shutdown()
```

Comment:
can we move these to a common utility for the tests? Or perhaps `events.go` in [utils](https://github.com/kubernetes-sigs/descheduler/tree/master/pkg/utils) so descheduler and tests can call it.

Reply:
Moved to events.go in utils. We still need to defer Shutdown in the upper level though, so I just created a function that returns both the broadcaster and the recorder.